src/main/java/org/spongepowered/common/config/category/TimingsCategory.java

```diff
@@ -33,22 +33,22 @@
 @ConfigSerializable
 public class TimingsCategory extends ConfigCategory {
 
-    @Setting
+    @Setting(comment = "???")
     private boolean verbose = false;
 
-    @Setting
+    @Setting(comment = "???")
     private boolean enabled = true;
 
-    @Setting("server-name-privacy")
+    @Setting(value = "server-name-privacy", comment = "???")
     private boolean serverNamePrivacy = false;
 
-    @Setting("hidden-config-entries")
+    @Setting(value = "hidden-config-entries", comment = "???")
     private List<String> hiddenConfigEntries = Lists.newArrayList("sponge.sql");
 
-    @Setting("history-interval")
+    @Setting(value = "history-interval", comment = "???")
     private int historyInterval = 300;
```

Comment:
ticks % history-interval
modulo how often a tick should be pushed to the timings history, not sure exactly what the history is